xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -295,6 +295,15 @@
                 #end
                 </select>
                 </td>
+              #elseif("$!colprop.type" == 'listboolean' && $isFilterable)
```

Comment:
Question: why do you introduce a "listboolean" type instead of "boolean"? That would be more simple.